en/docs/references/concepts/authorization/resource-owner-grant.md

```diff
@@ -32,6 +32,9 @@ You will receive a response similar to the format below.
 {"token_type":"Bearer","expires_in":2510,"refresh_token":"5ba3dedc77581df5f84f9b228eef0b91","access_token":"ca19a540f544777860e44e75f605d927"}
 ```
 
-!!! info "Support for refresh token grant - Yes"
-	This grant type issues a refresh token which can be used to obtain new access tokens using the [refresh token grant](refresh-token-grant.md).
+!!! info "Support for refresh token grant"
+	This grant type issues a refresh token which can be used to obtain new access tokens using the [refresh token grant](../refresh-token-grant).
 
+!!! info "Related topics"
+        - [Guide: Password Grant](../../../../guides/access-delegation/password-grant)
+        - [Demo: Password Grant](../../../../quick-starts/password-playground)
```

Comment:
We have called it Quick Start in most documents I guess. Shall we make this consistent?